### DISCUSSION TITLE
Accessing Loop N Times index

### DISCUSSION THREAD

**Discussion by edamreed** [postmanlabs/postman-flows#361] [Q&A]
Hi, I would like to access the iteration variable of the Loop N Times block in a request, in this case as the page number.
I have {{pageno}} in the get request but cant figure out how to set pageno from the Loop N Times output.

Further Info, I did find an example of a block that assigns a value to a variable but don't seem to have that block in my list of blocks I can add to the flow.
I'm using the Desktop app rather than the Web version.

**Comment by ghost** [comment]
The block you found, was it the "Create Variables" block? I believe the "Create Variables" block has recently been replaced by the "Create Data" block. Maybe that will work for you?

**Reply by ghost** [reply]
So, maybe: Loop N Times > Create Data (set pageno to input value) > request (set {{pageno}} to input)?

**Reply by edamreed** [reply]
I think I was being a bit dense, you can just connect the Loop N Times output to the send request and use /variables

**Comment by edamreed** [comment]
Answered my own question, you can use /variables directly in the send request variables and it works ok.
